src/analyzer.js

```diff
@@ -22,9 +22,26 @@ const JAVASCRIPT_KEYWORDS = new Set([
   'try',
   'var',
   'while',
+  'in',
+  'new',
+  'case',
+  'null',
+  'true',
+  'void',
+  'with',
+  'false',
+  'delete',
+  'export',
+  'import',
+  'public',
+  'static',
+  'default',
+  'package',
+  'private',
+  'interface',
 ]);
 
-const FIRST_KEYWORD_ON_LINE_REGEXP = /^\s*\/\/.*?([a-zA-Z_]\w?)/;
+const FIRST_KEYWORD_ON_LINE_REGEXP = /^\s*\/\/.*?(\w+)/;
```

Comment:
Thank you, that is indeed a bug. However, a keyword or identifier cannot start with a digit. Therefore I would prefer to use this:

```suggestion
const FIRST_KEYWORD_ON_LINE_REGEXP = /^\s*\/\/.*?([a-zA-Z_]\w*)/;
```

Reply:
That's correct, I missed that part, thank you for the correction.